site/docs/v1/tech/apis/_integration-request-body.adoc

```diff
@@ -25,22 +25,32 @@ True if the Kafka integration is enabled.
 [field]#integration.kafka.producer# [type]#[Map<String, String>]# [optional]#Optional#::
 String key value pairs to be used when building the Kafka Producer.
 
-[field]#integration.twilio.accountSID# [type]#[String]# [optional]#Optional#::
-The Twilio Account ID to use when connecting to the Twilio API. This can be found in your Twilio dashboard. Required when the Twilio integration is enabled.
+[field]#integration.twilio.accountSID# [type]#[String]# [optional]#Optional# [deprecated]#Deprecated#::
+blah blah blahThe Twilio Account ID to use when connecting to the Twilio API. This can be found in your Twilio dashboard. Required when the Twilio integration is enabled.
```

Comment:
I don't think you want 'blah blah blah' here. :)

Reply:
Oops! This was when I was getting frustrated when the page wasn't re-compiling.  Hence the slack convo where I learned I need to touch the file that includes this one! Funny...